script/urlChecker.ts

```diff
@@ -38,15 +44,17 @@ async function checkPages(bookSlug: string, pages: string[]) {
   const notFound: string[] = [];
 
   const visitPage = async(page: string) => {
+    const validatedUrl = getPageOrRedirectedUrl(redirects, page);
     try {
-      const response = await fetch(`${rootUrl}${page}`);
+      const response = await fetch(`${rootUrl}${validatedUrl}`);
```

Comment:
the criteria should be
1) if the page url returns a 200 on prod, 👍 
2) if the page url is a redirect target, one of the redirects returns a 200 on prod, 👍